src/query/api/v1/handler/prometheus/native/read_instantaneous_test.go

```diff
@@ -159,7 +170,8 @@ func testPromReadInstantHandler(
 			]
 		}
 	}
-	`, at0.Unix(), value0, at1.Unix(), value1))
+	`, jsonWarnings, at0.Unix(), value0, at1.Unix(), value1))
```

Comment:
Maybe we should get rid of the fmt.Sprintf stuff and introduce types `JSONMap`, `JSONArray` and then add a `MustPrettyJSONObject(t, ....)`.

I prototyped this here, looks easy to drop in I think?
https://play.golang.org/p/sTatAL6vaEE

Reply:
Hah, nice :)

Reply:
Actually, wouldn't using maps lead to mismatches in data? Also, just from my experience doing it in the last few minutes, building these structures seems more error prone than just writing up the expected JSON?

Reply:
If we MustPrettyJSON both of them, then they should get the correct ordering as per JSON pretty printing behavior.